packages/bootstrap/scss/drawer/_layout.scss

```diff
@@ -15,6 +15,12 @@
                     padding-left: $drawer-item-padding-x;
                 }
             }
+
+            // Fallback styles when drawer item is missing k-spacer element
+            .k-i-arrow-chevron-up,
+            .k-i-arrow-chevron-down {
```

Comment:
Targeting specific icon classes is not a good idea and flaky. Find another way.

Reply:
if this is "unification" the class must be present!